src/20240324_Multi_GeneralizedLTLTVReductionOnAave/GeneralizedLTLTVReductionOnAave.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Generalized LT/LTV Reduction on Aave"
+author: "Chaos Labs"
+discussions: "https://governance.aave.com/t/generalized-lt-ltv-reduction-on-aave/16766"
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0x4067d91ef5864925136d10ec9419f032a70f7e6489740386e348488426656274"
+---
+
+## Simple Summary
+
+Reduce stablecoin LTs and LTVs across all markets.
+
+## Motivation
+
+In response to the volatility risks highlighted by the UNI incident on Compound, this proposal aims to mitigate theoretical debt asset volatility by proposing a reduction in collateral asset borrowing power for stablecoins. While the UNI debacle was attributed to inadequate dual-bounded fallback oracle logic, a property not employed by Aave, leading to mispriced UNI debt and subsequent arbitrage activities, the observed upward volatility in longer-tailed assets suggests a prevailing “risk-on” market sentiment. Therefore, we advocate for minimizing borrowing power concerning shorting these assets to prevent potential short squeezes and liquidation shortfalls. Subsequent recommendations will address borrow cap reductions for relevant long-tailed assets, which generate minimal revenue relative to the theoretical risks, especially in edge-case scenarios. Additionally, presently only 2.4%, 3.7% and 5.8% of supplied DAI, USDT and USDC on Ethereum are being utilized as collateral, indicating the generalized lack of demand (and thus revenue).
+
+## Specification
+
+| Assets            | LT           | LTV       | Total Liquidated |
+| ----------------- | ------------ | --------- | ---------------- |
```

Comment:
Table should be more clear of which configs of tokens on which networks have been updated. As if not i would not know just by reading the table, which are on ethereum or arbitrum etc...